docs/styles.scss

```diff
@@ -4,7 +4,7 @@ $fa-font-path: '../node_modules/font-awesome/fonts';
 $FontPathOpenSans: '../node_modules/npm-font-open-sans/fonts';
 @import '../node_modules/npm-font-open-sans/open-sans';
 
-@import 'cashmere';
+@import '../lib/src/sass/cashmere';
```

Comment:
I am ok with this change but the reason it was working is the `includePath` in the config
https://github.com/angular/angular-cli/wiki/stories-global-styles

Reply:
This will be changed in the CLI update